### PR TITLE
Ava UI: Update Ava

### PR DESCRIPTION
**Pull request by IsaacMarovitz**
**Changes:**
- Update to Ava 11.0.10
- Remove System.Drawing.Common

### CODE REVIEW

File: Directory.Packages.props [hunk 1]
```diff
@@ -45,9 +45,8 @@
     <PackageVersion Include="SixLabors.ImageSharp" Version="1.0.4" />
     <PackageVersion Include="SixLabors.ImageSharp.Drawing" Version="1.0.0-beta11" />
     <PackageVersion Include="SPB" Version="0.0.4-build32" />
-    <PackageVersion Include="System.Drawing.Common" Version="8.0.2" />
     <PackageVersion Include="System.IO.Hashing" Version="8.0.0" />
     <PackageVersion Include="System.Management" Version="8.0.0" />
     <PackageVersion Include="UnicornEngine.Unicorn" Version="2.0.2-rc1-fb78016" />
   </ItemGroup>
-</Project>
+</Project>
```
Comment (TSRBerry) [inline Directory.Packages.props:52]: Missing new line

File: src/Ryujinx/Ryujinx.csproj [hunk 2]
```diff
@@ -55,9 +55,6 @@
     <PackageReference Include="SPB" />
     <PackageReference Include="SharpZipLib" />
     <PackageReference Include="SixLabors.ImageSharp" />
-
-    <!--NOTE: DO NOT REMOVE, THIS IS REQUIRED AS A RESULT OF A TRIMMING ISSUE IN AVALONIA -->
-    <PackageReference Include="System.Drawing.Common" />
```
Comment (TSRBerry) [inline src/Ryujinx/Ryujinx.csproj:60]: How did you test that this didn't break trimming again?

**Review by TSRBerry — COMMENTED** [review]
Lgtm, just commented one question.

File: src/Ryujinx/Ryujinx.csproj (hunk 2 above)
Comment (IsaacMarovitz) [reply src/Ryujinx/Ryujinx.csproj:60]: Avalonia no longer uses System.Drawing.Common, so there is no risk of breaking trimming.

Comment (TSRBerry) [reply src/Ryujinx/Ryujinx.csproj:60]: Awesome, thanks!

**Review by TSRBerry — APPROVED** [review]
Lgtm! I'd like to merge #6441 before we merge this though.

**Review by AcK77 — APPROVED** [review]

**Pull request merged**